modules/messages/src/lib.rs

```diff
@@ -1700,8 +1706,8 @@ mod tests {
 			.actual_weight
 			.unwrap();
 			assert!(
-				post_dispatch_weight.proof_size() < pre_dispatch_weight.proof_size(),
-				"Expected post-dispatch PoV {} to be less than pre-dispatch PoV {}",
+				post_dispatch_weight.proof_size() <= pre_dispatch_weight.proof_size(),
```

Comment:
Why this is changed? The test is called `proof_size_refund_from_receive_messages_proof_works` - it should test that the `post_dispatch_weight.proof_size()` is less than the `pre_dispatch_weight.proof_size()`. Otherwise, we should remove it :)

Reply:
Sorry, you're right. It was related to the fact that we didn't take into account the number of messages here. Since before it didn't matter. Fixed.